release notes/upcoming.md

```diff
@@ -13,3 +13,5 @@ Description of feature.
 * JS: Consistently report setup/teardown timeouts as such and switch the error message to be more
   expressive (#890)
 * JS: Correctly exit with non zero exit code when setup or teardown timeouts (#892)
+* Cloud: Correctly report thresholds who have failed during the execution but are actually not
```

Comment:
It is a bit unclear that this fix concerns the cloud output (`k6 run -o cloud script.js`) rather than the cloud execution (`k6 cloud script.js`)
Suggestion: `Thresholds: When outputting metrics to the Load Impact cloud, fix the incorrect reporting of threshold statuses at the end of the test.`